### PR TITLE
.git: add more skip words

### PR DESCRIPTION
**Pull request by tchaikov**
these words are either

* shortened words: strategy => strat, read_from_primary => fro
* or acronyms: node_or_data => nd

before we rename them with better names, let's just add them to the ignore word list.

### CODE REVIEW

**Comment by tchaikov** [comment]
this change conflicts with #16992. i will rebase it after #16992 lands.

**Review by denesb — APPROVED** [review]

**Comment by tchaikov** [comment]
@scylladb/scylla-maint hello maintainers, could you help queue this change?

**Pull request closed without merging**
